Sources/Commands/Options.swift

```diff
@@ -315,19 +315,20 @@ public struct SwiftToolOptions: ParsableArguments {
         archs.count > 1 ? .xcode : _buildSystem
     }
     
-    /// Tells `Workspace` to attempt to locate .netrc file at `NSHomeDirectory`.
-    @Flag()
-    var netrc: Bool = false
+    /// Whether to load .netrc files for authenticating with remote servers
+    /// when downloading binary artifacts or communicating with a registry.
+    @Flag(inversion: .prefixedNo, help: "Load credentials from a .netrc file")
```

Comment:
I other cases I seen SwiftPM use the prefix "disable/enable". is there a convention we want to follow here @abertelrud @neonichu?

Reply:
What I found with the that inversion option is that it doesn't, by itself, provide backwards compatibility with our existing `--netrc` option. To use `--enable-netrc` / `--disable-netrc`, we'd need to add it as a new flag, and make `--netrc` hidden. The issue there is that it's confusing how those flags should interact:

| `--netrc` | `--enable-netrc` | `--disable-netrc` | Outcome                         |
|-----------|------------------|-------------------|---------------------------------|
|           |                  |                   | Enabled                         |
| ✅         |                  |                   | Enabled with warning            |
| ✅         | ✅                |                   | Enabled with warning(s)         |
|           | ✅                | ✅                 | Error                           |
| ✅         |                  | ✅                 | Disabled with warning or error? |
|           | ✅                | ✅                 | Error                           |
|           |                  | ✅                 | Disabled                        |

Whereas for `--no-netrc`:

| `--netrc` | `--no-netrc` | Outcome                         |
|-----------|--------------|---------------------------------|
|           |              | Enabled                         |
| ✅         |              | Enabled with warning            |
|           | ✅            | Disabled                        |
|  ✅        | ✅            | Error                        |

Reply:
Note that the default behavior for flags is to use the last one provided, not to error if both show up. You can add a `exclusivity: .exclusive` parameter if you want `command --netrc --no-netrc` to result in an error.

Reply:
I do not have a strong opinion on this. @neonichu @abertelrud do you?

Reply:
we could emit a warning manually for the deprecated options like we do in: https://github.com/apple/swift-package-manager/blob/main/Sources/Commands/SwiftTestTool.swift#L538